x-pack/solutions/security/plugins/cloud_security_posture/server/routes/graph/parse_records.ts

```diff
@@ -159,23 +159,24 @@ const determineEntityNodeShape = (
 ): {
   shape: EntityNodeDataModel['shape'];
   icon?: string;
+  tag?: 'host' | 'user' | 'other types';
 } => {
-  // If actor is a user return ellipse
+  // If actor is a user return ellipse icon and user tag
   if (users.includes(actorId)) {
-    return { shape: 'ellipse', icon: 'user' };
+    return { shape: 'ellipse', icon: 'user', tag: 'user' };
```

Comment:
incase you go this route, `tag` should be in a Title casing

And please make sure to add a relevant UT for this